apps/web/pages/v2/event-types/index.tsx

```diff
@@ -609,6 +614,22 @@ const EventTypesPage = () => {
                   className="mb-4"
                 />
               )}
+              {!defaultCalendarConnected && (
+                <Alert
+                  severity="warning"
+                  className="mb-4"
+                  title={<>{t("missing_connected_calendar") as string}</>}
+                  message={
+                    <Trans i18nKey="connect_your_calendar_and_link">
+                      You can connect your calendar from
+                      <a href="/apps/categories/calendar" className="underline">
+                        here
+                      </a>
+                      .
+                    </Trans>
+                  }
+                />
+              )}
               {data.eventTypeGroups.map((group, index) => (
```

Comment:
this didnt seem to work in the old PR and was reverted, are we sure we wanna keep this?

Reply:
@alannnc  - Is it worth removing this?

Reply:
Just take whats on main and you should be ready to go. I think on main its called as <NoCalendarConnectedAlert/>